src/components/pageIntro/pageIntro.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import PageIntro from './PageIntro'
+
+storiesOf('Page Intro', module).add('default', () => (
+  <PageIntro
+    cta={{
+      link: { to: '/relative-url', text: 'Lorem ipsum' },
+      title: 'Lorem ipsum',
+      body:
+        'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.',
+    }}
+  >
+    {'Lorem Ipsum'}
```

Comment:
Could we make these editable (ie via `text` from `@storybook/addon-knobs`) so that we may edit the text from within Storybook? Makes it more dynamic and will be good once we get it published for Content/Designers. There should be a few examples of this in other stories!

Reply:
Added text knob